script.js

```diff
@@ -0,0 +1,28 @@
+let menu = document.querySelector(".menu")
+let ham = document.querySelector(".ham")
+let xIcon = document.querySelector(".xIcon")
+let menuIcon = document.querySelector(".menuIcon")
+
+
+
+function toggleMenu() {
+  if (menu.classList.contains("showMenu")) {
+    menu.classList.remove("showMenu");
+    xIcon.style.display = "none";
+    menuIcon.style.display = "block";
+  } else {
+    menu.classList.add("showMenu");
+    xIcon.style.display = "block";
+    menuIcon.style.display = "none";
+  }
+}
+
```

Comment:
- [ ] Kindly change this to use the es6 arrow function syntax following the JavaScipt best practices guide.